app/src/lang/translations/en-US.yaml

```
@@ -961,6 +961,8 @@ wysiwyg_options:
   source_code: Edit Source Code
   fullscreen: Full Screen
   directionality: Directionality
+  lazy_loading: Lazy Load Image
+  lazy_loading_label: Enable lazy loading
```

Would look a bit cleaner IMO:
```suggestion
  lazy_loading_label: Enable Lazy Loading
```